src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -26616,7 +26616,7 @@ private static final void registerOpticalComponentRecipes() {
                     245_760,
                     true);
             GT_Values.RA.addMultiblockChemicalRecipe(
-                    new ItemStack[] {Materials.Zeolite.getDust(0), GT_Utility.getIntegratedCircuit(1)},
+                    new ItemStack[] {GT_Utility.copyAmount(0, Materials.Zeolite.getDust(5), GT_Utility.getIntegratedCircuit(1))},
```

Comment:
okay it appears i found the problem.
GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Zeolite, 0, false) will fix it